### ISSUE TITLE
pdf utils improve linearized support

### ISSUE THREAD

**Issue by pofider** [jsreport/jsreport#1014]
This particular pdf can't be appended
https://web.wpi.edu/Images/CMS/Provost/landscape.pdf

https://forum.jsreport.net/topic/2662/howto-append-existing-pdf-to-a-generated

**Comment by fhrtms** [comment]
If i run it through [https://cloudconvert.com](https://cloudconvert.com) it works.

https://avepdf.com/de/pdfa-validation gives out a few errors:

![image](https://user-images.githubusercontent.com/5249946/208081401-04fa83b7-3e0e-405a-bf06-393725cd9261.png)

**Comment by fhrtms** [comment]
I can also do it with ghostscript.

`gs -sDEVICE=pdfwrite -dEmbedAllFonts=true -dPrinted=true -dPDFACompatibilityPolicy=1 -dBATCH -dNOPAUSE -dSAFER -dPDFA -sOutputFile=landscape_out.pdf landscape.pdf`

**Comment by fhrtms** [comment]
With `npm i ghostscript-node` i get a valid PDF.

```javascript
const gs = require("ghostscript-node");

async function afterRender(req, res) {

    // load PDF as Buffer
    const pdf1 = await downloadPDF('https://www.aubrete.lt/wp-content/uploads/2019/06/1-COVERSTYLE-1.pdf')

    // get number of pages of pdf1
    const numPagesPDF1 = await gs.countPDFPages(pdf1);

    // get Buffer object with bytes of specified set of pages
    // page numbers begin with 1, last page is included
    const partOfPDF1 = await gs.extractPDFPages(pdf1, 1, numPagesPDF1);

    // checks if pdf1 is a valid PDF file
    const isPDF1Valid = await gs.isValidPDF(pdf1);
    
    if(isPDF1Valid) {
        res.content = await jsreport.pdfUtils.append(res.content, partOfPDF1)
    }
}
```

**Comment by pofider** [comment]
It's fixed now, it will be available in the jsreport 3.11.0
